src/frontends/onnx/frontend/include/onnx_import/core/node.hpp

```diff
@@ -32,10 +32,10 @@ namespace ngraph {
 namespace onnx_import {
 namespace error {
 namespace node {
-struct UnknownAttribute : ngraph_error {
+struct UnknownAttribute : ov::Exception {
     OPENVINO_SUPPRESS_DEPRECATED_START
     explicit UnknownAttribute(const std::string& node, const std::string& name)
-        : ngraph_error{"Node (" + node + "): unknown attribute \'" + name + "\'"} {}
+        : ov::Exception{"Node (" + node + "): unknown attribute \'" + name + "\'"} {}
```

Comment:
Please, refactor exception to use `create()`. As reference, see `ov::Cancelled`

Reply:
As we discussed offline, looks like it is meaningless exceptions (aren't catching and handling above in stack). I've refactored this part of code and bring more clarity into error messages.